package.json

```
@@ -14,12 +14,15 @@
     "@angular/cdk": "^12.1.4",
     "@angular/common": "~12.1.0",
     "@angular/compiler": "~12.1.0",
+    "@angular/elements": "^12.1.4",
     "@angular/core": "~12.1.0",
     "@angular/forms": "~12.1.0",
+    "@angular/google-maps": "^10.2.7",
     "@angular/material": "^12.1.4",
     "@angular/platform-browser": "~12.1.0",
     "@angular/platform-browser-dynamic": "~12.1.0",
     "@angular/router": "~12.1.0",
+    "@angular/youtube-player": "^10.2.7",
```

issue: Use the same version across Angular Components packages. Also update `package-lock.json` after making this change.
```suggestion
    "@angular/google-maps": "^12.1.4",
    "@angular/material": "^12.1.4",
    "@angular/platform-browser": "~12.1.0",
    "@angular/platform-browser-dynamic": "~12.1.0",
    "@angular/router": "~12.1.0",
    "@angular/youtube-player": "^12.1.4",
```